config/ldap_publick8s.yaml

```diff
@@ -0,0 +1,46 @@
+service:
+  type: LoadBalancer
```

Comment:
A thought about this LoadBalancer: the helm chart allows specifying a predefined IP through the `service.IP` value (ref. https://github.com/jenkins-infra/helm-charts/blob/4dc496fe3f2d17f399272ab6fdd6903083031089/charts/ldap/values.yaml#L58).

- You could define an Azure Public IP resource in jenkins-infra/azure which value could be specified here. Then, you could use jenkins-infra/azure (instead of azure-net) to have a DNS record which value is set from the IP.
  - This case makes the IP deterministic but MUST be done before the initial creation of the LB, because it cannot be changed later.
- Otherwise you'll have to wait for the release to be installed and the LB allocated, to retrieve the IP of the LB adn set the DNS `A` record (maybe a data source in terraform?)

Reply:
To be revised in another context than the migration one, WDYT?

Reply:
as seen together: first choice (merge, let the LB be created, retrieve its IP and add it in the DNS record )

Reply:
IP created in https://github.com/jenkins-infra/azure/pull/400

Reply:
Addressed in [`8db9381` (#4044)](https://github.com/jenkins-infra/kubernetes-management/pull/4044/commits/8db9381f9ae318998e6c6aca6ab2ab3acdd0221d)